_i18n/de/resources/user-guides/verification-allos-advanced.md

```diff
@@ -84,39 +84,39 @@ gpg:               imported: 1
 gpg: marginals needed: 3  completes needed: 1  trust model: pgp
 ```
 
-If you have imported the key previously, the output will look like this:
+Falls du den Schlüssel zuvor bereits importiert hast, wird der Output folgendermaßen aussehen:
 
 ```
 gpg: key F0AF4D462A0BDF92: "binaryFate <binaryfate@getmonero.org>" not changed
 gpg: Total number processed: 1
 gpg:              unchanged: 1
 ```
 
-## 3. Download and Verify Hash File
+## 3. Download und Verifikation der Hash-Datei
 
-This section will cover downloading the hash file and verifying its authenticity.
+Der folgende Abschnitt behandelt den Download und die Echtheitsverifizierung der Hash-Datei.
 
-### 3.1. Get Hash File
+### 3.1. Hash-Datei herunterladen
 
-On Windows or Mac, go to the [hashes file on getmonero.org]({{ site.baseurl_root }}/downloads/hashes.txt) and save the page as `hashes.txt` to your home directory.
+Auf Windows oder Mac: Gehe zur [Hashes-Datei auf getmonero.org]({{ site.baseurl_root }}/downloads/hashes.txt) und speichere die Seite unter `hashes.txt` in deinem "Home"-Verzeichnis.
```

Comment:
In some other places you replaced baseurl_root with baseurl. Here you didn't replace it. Not sure which is correct?

Reply:
This one should be correct. :)